### PR TITLE
Update code snippets in quickstart, and copyright headers

### PR DESCRIPTION
**Pull request by mgorny**
1. Update copyright header to state 'Gentoo Authors' everywhere.
2. Update the code snippets in quickstart to be more modern:
   * use eapply instead of epatch
   * avoid redefining default src_install whenever it would just work
   * prefer `myconf` array over backslash hell for large number of flags
3. Correct licenses

### CODE REVIEW

File: quickstart/text.xml [hunk 1]
```diff
@@ -397,20 +381,23 @@ DEPEND="${RDEPEND}
...
+		$(use_enable ssl)
+		$(use_enable crypt gpgme)
+		$(use_enable pda jpilot)
+		$(use_enable ldap)
+		$(use_enable ipv6)
+		$(use_enable imlib)
 		$(use_enable xface compface)
+	)
+
+	econf "${myconf[@]}"
```
Comment (ulm) [inline quickstart/text.xml]: That unnecessarily introduces a variable, and I think it is a matter of personal preference.
Also the existing example is easier to read (IMHO), so please leave it as it is.

Comment (mgorny) [reply quickstart/text.xml]: There is no harm in using a local variable. Relying on backslashes is harmful, especially when someone accidentally forgets a backslash in middle of it. And yes, I've seen such mistakes in practice.

Comment (mgorny) [reply quickstart/text.xml]: Also, case of passing flags directly to econf is presented in another example. I think it's beneficial to present more than one possible solution.

Comment (ulm) [reply quickstart/text.xml]: I disagree. This is just your personal preference, not something that needs to be recommended in general. That there can be unrelated mistakes is not an argument. The variable could be misspelt as well, or the quotes be omitted or whatnot.

Comment (mgorny) [reply quickstart/text.xml]: Then just skip that one commit.

**Comment by ulm** [comment]
OK, if there are no further comments until tomorrow, I am going to merge this series, except for commit 
f1fafdc .

**Comment by ulm** [comment]
Cherry-picked commits 1, 2, 3, and 5. Closing.

**Pull request closed without merging**